core/dbt/events/types.py

```diff
@@ -1928,8 +1928,8 @@ class PrintHookStartLine(InfoLevel, Cli, File, NodeInfo):
     index: int
     total: int
     truncate: bool
-    report_node_data: Any  # TODO use ParsedHookNode here
-    code: str = "Z032"
+    report_node_data: ParsedHookNode
```

Comment:
I couldn't figure out how to get these `mypy` checks passing either!
```
core/dbt/task/run.py:361: error: Argument "report_node_data" to "PrintHookStartLine" has incompatible type "dbt.contracts.graph.parsed.ParsedHookNode"; expected "dbt.events.stubs.ParsedHookNode"
```

Reply:
Take a close look at that error. There's two `ParsedHookNode`s: one is a type stub and one is a real class. We need to make sure we're not using the type stub, and delete it if we can.

Reply:
So I am using the stub type.  **But** I"m also referencing `ParsedHookNode` on other events that don't trigger the error.  Then I'm using the stub type for `ParsedModelNode` on 16 events that don't trigger this.  Why does it only trigger sometimes?

Reply:
probably because mypy doesn't run on all the files so some of the call sites don't get caught.